.github/workflows/build_and_test.yml

```diff
@@ -41,11 +42,13 @@ jobs:
       with:
         python-version: '3.9'
     - name: Build and test including remote checks (3.9) mypy
+      shell: bash
       if:  (matrix.os == 'macos-11') && ((github.event_name == 'pull_request' &&  github.event.pull_request.head.repo.full_name == github.repository) || github.event_name == 'release' || contains(github.ref, 'refs/heads/wheel'))
       run: |
         ./.github/workflows/build-test mypy
     - name: Build and test including remote checks (3.9) nomypy
       if:  (matrix.os != 'macos-11') && ((github.event_name == 'pull_request' &&  github.event.pull_request.head.repo.full_name == github.repository) || github.event_name == 'release' || contains(github.ref, 'refs/heads/wheel'))    
+      shell: bash
```

Comment:
This is something we need to change on pytket-quantinuum, iqm, qujax, pennylane and braket, too?

Reply:
This fix has already been made to some of them, though perhaps not all.